### PR TITLE
Document example of extraVolumes

### PR DESCRIPTION
**Pull request by manics**
See https://github.com/jupyterhub/zero-to-jupyterhub-k8s/issues/1109

### CODE REVIEW

File: doc/source/user-storage.md [hunk 1]
```diff
@@ -220,3 +220,25 @@ kubectl --namespace=<your-namespace> delete pvc <pvc-name>
...
+```yaml
+singleuser:
+  storage:
+    extraVolumes:
+      - name: jupyterhub-shared
+        persistentVolumeClaim:
+          claimName: jupyterhub-shared-volume
+    extraVolumeMounts:
+      - name: jupyterhub-shared
+        mountPath: /home/jovyan/shared
```
Comment (consideRatio) [inline doc/source/user-storage.md]: @manics :heart: a documentation PR! I'm growing more and more fond of documentation as I have consumed so much lately.

---

I think this mount may fail in some way, as the `mountPath` is within another mount's path. If I recall correctly what I've read but not tested myself. Can one nest volumeMounts?

Perhaps we could also reference [k8s docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#volumemount-v1-core) for volumeMounts and/or provide a note about `subPath` for entries in `extraVolumeMounts` while we are at it.

Comment (consideRatio) [reply doc/source/user-storage.md]: If it is possible to nest volumes like this, it LGTM!

Comment (manics) [reply doc/source/user-storage.md]: Good point, on my system there aren't any per-user volumes. How does `/home/shared` sound?

Comment (consideRatio) [reply doc/source/user-storage.md]: Sounds great @manics !

Comment (manics) [reply doc/source/user-storage.md]: Updated!

**Comment by manics** [comment]
Looks like travis failed with a transient error: `curl: (7) Failed to connect to storage.googleapis.com port 443: Connection timed out`, can you retrigger the job?

**Pull request merged**